client/plots/profilePlot.js

```diff
@@ -630,3 +629,12 @@ export function getDefaultProfilePlotSettings() {
 		showTable: true
 	}
 }
+
+export function getProfileLogin() {
+	const queryString = window.location.search
+	const urlParams = new URLSearchParams(queryString)
+	const logged = urlParams.get('logged')
+	const site = urlParams.get('site')
+	const user = logged && site ? 'site' : logged ? 'admin' : 'public'
+	return [logged, site, user]
```

Comment:
I like this function. Later on when the actual sign-in mechanism is available, it would make is easy to replace the code here to use `app.vocabApi()` to get the logged-in status/role.

Reply:
Also, could do this later, not urgent: not having a user should be equivalent to not being logged in, I don't think there is a need for a separate `logged` variable.

Reply:
Right now _public_ is a user. Yeah we can review this later.